client/app/services/query.js

```diff
@@ -115,7 +115,7 @@ export class Parameter {
       }
 
       if (isArray(value) && (value.length === 2)) {
-        value = [moment(value[0]), moment(value[1])];
+        value = [moment(value[0], 'YYYY-MM-DD'), moment(value[1], 'YYYY-MM-DD')];
```

Comment:
What does it fix?

Reply:
It fixes a warning message from Moment (it now requires you to supply the date format without relying on the defaults).